### ISSUE TITLE
A wonderful, wonderful addon! 

### ISSUE THREAD

**Issue by tohtorizorro** [igrigorik/videospeed#519]
I use Video Speed Controller features constantly when watching any video content in browser (youtube, netflix, hbo, etc) and just wanted to stop by and tell how much I appreciate your work guys!

I Originally installed the addon for online studying to make unbearably slow paced or verbose video tutorials watchable and it has been totally excellent for that. Nowadays with pure talk content I tend to keep the speed mostly at 1.30 - 1.80. Also with services like HBO who can't seem to manage include time skip buttons (let alone hotkeys) it's nice to be able to rewind in just the amount you want with just the key you want. On that note, it would be awesome if play/pause and volume controls bindings could be assigned with VSC (like in the advanced settings or so). 

The functinality VCS offers would benefit so many it should be a feature inluded in Chrome by default but I don't see this happening any time soon so THANK YOU once more for delivering it!!!

BR,
Mikko from Finland

**Comment by jacobcolbert** [comment]
I totally agree that VCS is life changing! HTt is why i started contributing to it! You already can set pause/play binding in the advanced settings. Volume controls seem beyond the scope of the project. Thanks for your note.

**Comment by tohtorizorro** [comment]
Thanks for pointing out that the pause-feature can be added to the list of shortcut keys. I just verified it works on my win & osx setups perfectly, even assigning spacebar works without interfering the functionality of players which already have it assigned to pause.

In the sense that the promised feature of the extension is to allow video speed control I agree that volume control is out of scope. I just figured (admitting being no expert in the subject) that since the app already works so universally (like even with hbo which I believe still use flash), adding volume control could've been a relatively small effort, and would nicely fit the UI already (adding the +/- shortcuts to the list). Then the extenaion could be named Video Total Control :)

Anyways, I'd be happy to donate a little something for application I use on daily basis. Is there some official means to do so?

Cheers!
